org.gnome.Podcasts.json

```diff
@@ -8,17 +8,11 @@
     ],
     "command" : "gnome-podcasts",
     "finish-args" : [
-        "--filesystem=xdg-run/dconf",
-        "--filesystem=~/.config/dconf:ro",
-        "--talk-name=ca.desrt.dconf",
-        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
         "--share=network",
         "--share=ipc",
-        "--socket=x11",
+        "--socket=fallback-x11",
         "--socket=wayland",
-        "--socket=pulseaudio",
-        "--device=dri",
-        "--own-name=org.mpris.MediaPlayer2.Podcasts"
```

Comment:
Just confirming upstream renamed this?

Reply:
Yeap, here's the upstream [MR](https://gitlab.gnome.org/World/podcasts/merge_requests/124)

Even mentions your blog post :P